### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by pojith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# <span id="N4Mintroduction">NFDI4Microbiota - Metadata Standards</span> <br><br>
+NFDI4Microbiota - Metadata Standards
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />
-<br><br> This work is licensed under a
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative
-Commons Attribution 4.0 International License</a>.
+This work is licensed under a
```
Comment (cpauvert) [inline README.md:4]: Thanks for starting to cleanup but you went a bit too far and removed the URL to the license. Below is a suggestion on how to do such a link using the Markdown syntax.

```suggestion
This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+Creative
```
Comment (cpauvert) [inline README.md:5]:
```suggestion
```
put on a single line

File: README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+Commons Attribution 4.0 International License.
```
Comment (cpauvert) [inline README.md:6]:
```suggestion
```
put on a single line

File: README.md [hunk 4]
```diff
@@ -703,9 +703,6 @@ Tatusova, et al. 2008. “The Minimum Information about a Genome Sequence
 “GSC MIxS: Host-associatedMIMS.”
 <https://genomicsstandardsconsortium.github.io/mixs/Host-associatedMIMS/>.
 
-“GSC MIxS: Human-associatedMIMS.”
```
Comment (cpauvert) [inline README.md:706]: Is there any reason why you removed this URL @pojith?

**Review by cpauvert — CHANGES_REQUESTED** [review]
Hi @pojith,
thanks for contributing to #20! I thought you wanted to start with #19 as you commented directly on this issue, and it is a good first issue for someone not familiar with the repository. 
That being said, any help is appreciated, I made a couple of comments regarding your contribution here that would need to be addressed before merging your PR. Let me know if something needs to be further explained.
Best,

File: README.md (hunk 4 above)
Comment (pojith) [reply README.md:706]: no

**Comment by pojith** [comment]
can you check now .

File: README.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
 # <span id="N4Mintroduction">NFDI4Microbiota - Metadata Standards</span> <br><br>
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />
-<br><br> This work is licensed under a
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative
-Commons Attribution 4.0 International License</a>.
++This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
+-Creative
```
Comment (cpauvert) [inline README.md:5]:
```suggestion

```

File: README.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
...
+-Commons Attribution 4.0 International License.
```
Comment (cpauvert) [inline README.md:6]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
 # <span id="N4Mintroduction">NFDI4Microbiota - Metadata Standards</span> <br><br>
 
 <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />
-<br><br> This work is licensed under a
-<a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative
-Commons Attribution 4.0 International License</a>.
++This work is licensed under a [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/).
```
Comment (cpauvert) [inline README.md:4]: Thanks! I removed the extra lines that must have slipped from a copy paste.

**Review by cpauvert — APPROVED** [review]

**Comment by cpauvert** [comment]
> can you check now .

Thanks for your contribution @pojith. We will add our code of conduct #23 to state how we want to maintain a friendly collaborative environment, but as a rule of thumb for the future, using "please" or "thanks" never hurt ;)

**Pull request merged**